draft-ietf-teep-architecture.md

```diff
@@ -1177,13 +1177,16 @@ from compromise by another user or tenant, even if the attacker has TAs.
 
 The protocol between TEEP Agents and TAMs similarly is responsible for
 securely providing integrity and confidentiality protection against
-adversaries between them. Since the transport protocol under the TEEP
-protocol might be implemented outside a TEE, as discussed in {{broker}},
-it cannot be relied upon for sufficient protection.  The TEEP protocol
-provides integrity protection, but confidentiality must be provided by
-payload encryption, i.e., using encrypted TA binaries and encrypted
-attestation information.  See {{I-D.ietf-teep-protocol}} for more
-discussion.
+adversaries between them. It is a design choice at what layers to best 
+provide protection against network adversaries. As discussed in {{broker}}, 
+the transport protocol and the Transport Layer Security protocol under 
```

Comment:
This wording implies that only TLS is legal under the TEEP protocol, which is not the case.  It's up to each transport protocol to specify.

```suggestion
the transport protocol and any security mechanism associated with it (e.g., the Transport Layer Security protocol) under 
```

Reply:
The change looks good to me.